### PR TITLE
added changes for the update package highlight.js 9.15.8

### PR DESCRIPTION
**Pull request by sumanprogress**
Signed-off-by: sumanprogress <jiddiSuman.Yadav@progress.com>

### :nut_and_bolt: Description: What code changed, and why?
We need to Upgrade highlight.js 9.15.8 or later

### :chains: Related Resources
#5179

### :+1: Definition of Done
I have updated the highlight.js package version.

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

**Review by vinay033 — APPROVED** [review]

**Comment by chaitali-mane** [comment]
Highlight.js must be added into devdependancy.

**Comment by chaitali-mane** [comment]
@sumanprogress 
please update 'highlight.js' in 'typedoc' of package-lock.json

File: components/chef-ui-library/package-lock.json [hunk 1]
```diff
@@ -8107,4 +8107,4 @@
       }
     }
   }
-}
+}
```
Comment (chaitali-mane) [inline components/chef-ui-library/package-lock.json:8110]: Add empty line at end of page

Comment (vinay033) [reply components/chef-ui-library/package-lock.json:8110]: @sumanprogress please take care of this next time, the same comment I have posted on your other PR.

**Review by vinay033 — CHANGES_REQUESTED** [review]

File: components/chef-ui-library/package-lock.json (hunk 1 above)
Comment (sumanprogress) [reply components/chef-ui-library/package-lock.json]: new line added

**Review by chaitali-mane — APPROVED** [review]

**Review by vinay033 — APPROVED** [review]

**Pull request merged**